javatests/arcs/showcase/typevariables/SkuRedactor.kt

```diff
@@ -0,0 +1,28 @@
+package arcs.showcase.typevariables
+
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.Job
+
+class SkuRedactor : AbstractSkuRedactor() {
+
+    /** Redact each input sku, and store the results to the output handle. */
+    override fun onStart() {
+        handles.input.onUpdate {
+            for (item in it.stream()) {
+                assertThat(item.sku).isNotEmpty()
```

Comment:
Isn't `assertThat` only for tests?

Reply:
These are kind of tests... But I would personally also do `require(item.sku.isNotEmpty())` or something along those lines.